R/functions-Spectrum.R

```diff
@@ -439,7 +439,7 @@ pickPeaks_Spectrum <- function(object, halfWindowSize = 2L,
 }
 
 smooth_Spectrum <- function(object,
-                            method = c("SavitzkyGolay", "MovingAverage"),
+                            method = c("SavitzkyGolay", "MovingAverage","MovingWeightedAverage"),
```

Comment:
Please add a space after the comma. We try to follow the Bioconductor coding style: https://bioconductor.org/developers/how-to/coding-style/